src/test/test-udev.c

```diff
@@ -113,9 +113,15 @@ static int run(int argc, char *argv[]) {
 
                         (void) mkdir_parents_label(devname, 0755);
                         (void) sd_device_get_devnum(dev, &devnum);
-                        assert_se(mknod(devname, mode, devnum) == 0);
+                        if (mknod(devname, mode, devnum))
+                        {
+                                return log_error_errno(errno, "mknod: '%s': %m", devname);
+                        }
```

Comment:
please follow coding style: no {} around single line if blocks, please. (And when: place { on same line as 'if' please).

For details see https://systemd.io/CODING_STYLE

Reply:
Oh, and we usually check return values of syscalls with `< 0`. i.e.

```c
if (mknod(…) < 0)
        return log_error_errno(errno, …);
```

Reply:
I've removed braces and updated return value checks.